src/rexsdesktop/model/ENV.java

```diff
@@ -20,7 +20,7 @@
     public static final String DB_HOST = "localhost";

```

Comment:
also this isn't the best way how to handle env variables, hopefully we could see this later